src/javascript/gameInstance.js

```diff
@@ -312,6 +321,14 @@ export default class GameInstance {
     this.holdingNote[key] = obj;
   }
 
+  seekTo(time) {
+    if (this.vm.srcMode === "youtube") {
+      return this.vm.ytPlayer.seekTo(Number(time));
+    } else {
+      return this.audio.seek(Number(time));
```

Comment:
Again, are these casts actually necessary?

Reply:
I'll try to remove them. I wish I could choose typescript...